readthedocs.yml

```diff
@@ -3,10 +3,11 @@ version: 2
 formats: all
 
 build:
-  image: latest
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.11"
```

Comment:
Let's hope this works

Reply:
Pretty sure it did. The rtd checks failed before this commit. I figured this was a convenient time to bump python from :gasp: 3.6